src/itemdef.cpp

```diff
@@ -169,10 +169,20 @@ void ItemDefinition::serialize(std::ostream &os, u16 protocol_version) const
 
 	os << serializeString16(short_description);
 
-	os << place_param2;
+	if (protocol_version <= 43) {
+		// Uncertainity whether 0 is the specified prediction or means disabled
+		if (place_param2)
+			os << place_param2.value();
+		else
+			os << (u8)0;
+	}
 
 	sound_use.serializeSimple(os, protocol_version);
 	sound_use_air.serializeSimple(os, protocol_version);
+
+	os << (u8)place_param2.has_value(); // protocol_version >= 43
+	if (place_param2)
+		os << place_param2.value();
```

Comment:
I wanted to have the value in the same place so that it would be easier to clean up the code in a few years when the protocol version support is dropped for this version.